### PR TITLE
Make Avatar work with Image component refactor

### PR DESCRIPTION
**Pull request by danoc**
Fixes https://github.com/thumbtack/thumbprint/issues/69.

**Todo:**
- [x] Remove lazysizes
- [x] Share size information across JS and SCSS
- [x] Wait until the recent Image component breaking changes are live in website.
- [x] Wait until #320 is merged.

### CODE REVIEW

**Comment by danoc** [comment]
Once #301 merges, we should change this to base off of `master`. It can be released independently of #301.

File: yarn.lock [hunk 1]
```diff
@@ -10199,11 +10199,6 @@ lazy-req@^1.1.0:
   version "1.1.0"
   resolved "https://registry.yarnpkg.com/lazy-req/-/lazy-req-1.1.0.tgz#bdaebead30f8d824039ce0ce149d4daa07ba1fac"
 
-lazysizes@^4.1.5:
```
Comment (tomgenoni) [inline yarn.lock]: nice

File: packages/thumbprint-react/components/Avatar/index.jsx [hunk 2]
```diff
@@ -148,59 +142,48 @@ EntityAvatar.defaultProps = {
...
+            {imageUrl ? (
+                <Image
+                    className={styles.circleAvatar}
+                    src={imageUrl}
+                    alt={fullName && `Avatar for ${fullName}`}
+                    title={fullName && `Avatar for ${fullName}`}
+                    height={dimensions[size]}
+                />
+            ) : (
+                <span
```
Comment (tomgenoni) [inline packages/thumbprint-react/components/Avatar/index.jsx]: Any reason this a `span` and not `div`?

**Comment by danoc** [comment]
@tomgenoni — This is ready for a final review.

**Pull request merged**